scheduling/taskmetadata_manager_test.go

```diff
@@ -708,7 +708,7 @@ func TestSetStatusByRunID(t *testing.T) {
 	}
 
 	fn := func(t *testing.T, test TestCase) {
-		manager := NewMemoryTaskMetadataManager()
+		manager, _ := NewMemoryTaskMetadataManager()
```

Comment:
don't drop

Reply:
accidentally, had addressed these in the RDS PR.